src/utils.ts

```diff
@@ -127,11 +127,9 @@ export function getFirstWorkspaceFolderPath(): string {
   if (!(vscode.workspace.workspaceFolders &&
```

Comment:
Same here to apply inverted into condition statement. Is uri.fsPath possible to be undefined?
```
if (!vscode.workspace.workspaceFolders || vscode.workspace.workspaceFolders.length === 0)
```

Reply:
Fixed.